code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -1,24 +1,25 @@
 #define EGG_INCUBATION_TIME 120
 
 /mob/living/simple_animal/hostile/headcrab
-	name = "headslug"
-	desc = "Absolutely not de-beaked or harmless. Keep away from corpses."
+	name = "head-spider"
+	desc = "An amalgamation of human and, well, something not human. it gazes intensely into the distance chittering it's teeth."
+	icon = 'icons/mob/changeling.dmi'
```

Comment:
Undeclared change. I kinda liked headslug as a name. :(

Reply:
this was undeclared, sorry. i'll fix that